src/EditorFeatures/Core/Implementation/Workspaces/EditorInfoBarService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
i would just make a top level folder called 'InfoBar' for this stuff.  I don't see why this is under Workspaces.